ChangeLog

```diff
@@ -3,21 +3,20 @@ Pylint's ChangeLog
 ------------------
 
 
-
 What's New in Pylint 2.10.0?
 ============================
 Release date: TBA
 
-* Added ``ignored-parents`` option to the design checker to ignore specific
-  classes from the ``too-many-ancestors`` check (R0901).
-
-  Partially closes #3057
-
```

Comment:
Maybe we could minimise the diff here so there is less conflict in other MR.

Reply:
I moved the Changelog style changes to #4770